examples/cluster/plot_hdbscan.py

```diff
@@ -213,3 +212,28 @@ def plot(X, labels, probabilities=None, parameters=None, ground_truth=False, ax=
     labels = hdb.labels_
 
     plot(X, labels, hdb.probabilities_, param, ax=axes[i])
+
+# %%
+# `dbscan_clustering`
+# ^^^^^^^^^^^^^^^^^^^
+# When fitting `HDBSCAN` we build a single-linkage tree which encodes the
+# clustering of all points across all values of the DBSCAN* `eps` parameter.
+# We can thus plot and evaluate these clusterings efficiently without fully
+# recomputing intermediate values such as core-distances, mutual-reachability,
+# and the minimum spanning tree. All we need to do is specify the `cut_distance`
+# (equivalent to `eps`) we want to cluster with, along with `min_cluster_size`,
+# the minimum number of points in a cluster below which the cluster is considered
+# noise.
+
+PARAM = (
+    {"cut_distance": 0.1, "min_cluster_size": 5},
+    {"cut_distance": 0.5, "min_cluster_size": 10},
+    {"cut_distance": 1.0, "min_cluster_size": 2},
```

Comment:
Can we simplify this example by only changing the `cut_distance`? I do not see any difference with keeping `min_cluster_size` as the default value.